sql/rpl_slave.cc

```diff
@@ -954,7 +984,12 @@ static int find_first_relay_log_with_rotate_from_master(Relay_log_info *rli) {
   char master_log_file[FN_REFLEN];
   my_off_t master_log_pos = 0;
 
-  if (channel_map.is_group_replication_channel_name(rli->get_channel())) {
+  if (channel_map.is_group_replication_channel_name(rli->get_channel())
+  #ifdef WITH_WSREP
+      // Similar to GR, do not allow operations on the wsrep channel
+      || wsrep_is_wsrep_channel_name(rli->get_channel())
+  #endif /* WITH_WSREP */
+    ) {
     LogErr(INFORMATION_LEVEL,
```

Comment:
Error message may create confusion since it would point to GR even though user end up using wsrep.
Can we segregate our code block along with custom error message based on how GR emits it.

Reply:
Check/error message separated out.